dataproc/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN apt update && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
     apt update && apt install -y google-cloud-sdk && \
     apt clean && rm -rf /var/lib/apt/lists/* && \
-    pip3 install hail==0.2.73
+    pip3 install cpg-hail
```

Comment:
Should we pin the version?

Reply:
Don't think it's really needed for the dataproc helper, it's fine to take the latest